### PR TITLE
Document the changes to the Activity app in ver 2.5.3

### LINKED ISSUE #2388: Activity app - Add a config option to send notifications ASAP
We need to document new setting and the new occ command.
https://github.com/owncloud/activity/pull/833

### PR DESCRIPTION
**Pull request by settermjd**
This PR:

- Updates the documentation for the activity configuration drop-down option in the user manual, documenting the new "_As soon as possible_" option 
- Documents the new activity occ command, `activity:send-emails`.

This fixes #2388.
This relates to https://github.com/owncloud/activity/pull/833.

### CODE REVIEW

File: modules/admin_manual/pages/configuration/server/app_commands/activity_commands.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+The `activity` commands are used for automating activity notifications in ownCloud server.
+
+----
+ activity
+  activity:send-emails  Send all pending activity emails now
+----
+
+== activity:send-emails
+
+The `activity:send-emails` command sends all pending activity emails as soon as possible.
```
Comment (VicDeo) [inline modules/admin_manual/pages/configuration/server/app_commands/activity_commands.adoc]: not 'as soon as possible' but immediately regardless the time they are scheduled

File: modules/user_manual/pages/apps/activity.adoc [hunk 2]
```diff
@@ -55,5 +55,5 @@ image::apps/activity/activity-settings-limit-to-favorites.png[Limit activity set
 
 === Configuring the Email Notification Interval
 
-In addition to enabling and disabling email notifications, bulk email notifications can be configured to be sent out hourly, daily, or weekly.
+In addition to enabling and disabling email notifications, bulk email notifications can be configured to be sent out: _As soon as possible_, _Hourly_, _Daily_, and _Weekly_.
```
Comment (VicDeo) [inline modules/user_manual/pages/apps/activity.adoc]: May be add a clarification that 'As soon as possible' means the next  cron execution?

**Comment by settermjd** [comment]
@VicDeo, thanks for providing feedback. I have addressed both points.

File: modules/admin_manual/pages/configuration/server/app_commands/activity_commands.adoc [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+The `activity` commands are used for automating activity notifications in ownCloud server.
+
+----
+ activity
+  activity:send-emails  Send all pending activity emails now
+----
+
+== activity:send-emails
+
+The `activity:send-emails` command sends all pending activity emails immediately, regardless the time they are scheduled.
```
Comment (phil-davis) [inline modules/admin_manual/pages/configuration/server/app_commands/activity_commands.adoc]:
```suggestion
The `activity:send-emails` command sends all pending activity emails immediately, regardless of the time they are scheduled.
```

**Comment by phil-davis** [comment]
@settermjd ready for backports

**Pull request merged**